src/src_V2/components/tooltip/TooltipText.jsx

```diff
@@ -0,0 +1,19 @@
+import { useRef } from 'react';
+import { Tooltip } from '@mui/material';
+
+import { useHoverTooltip } from '../../hooks';
```

Comment:
Also i just noted tha hook folder in the global scope i think it is kinda util or somethihng like that and don't need to be in the app level folder(src/src_v2)

Reply:
Is it okay to locate the hooks folder in `src_V2/modules/global/hooks/filename.js` as Tigran did?